recipes/pycwt/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "pycwt" %}
+{% set version = "0.3.0a22" %}
+{% set sha256 = "f99668ef2caf8d5020ef6bcbbcc86ee542d038ae7c5e1e80da12e5264445eb17" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
```

Comment:
It looks like only `python` and `setuptools` are required to build the package since we're using `--single-version-externally-managed`